xml/admin_rbd.xml

```diff
@@ -155,135 +155,177 @@
-       </para>
+     <screen>&prompt.root;mkfs.xfs /dev/rbd0
+      log stripe unit (4194304 bytes) is too large (maximum is 256KiB)
+      log stripe unit adjusted to 32KiB
+      meta-data=/dev/rbd0              isize=256    agcount=9, agsize=261120 blks
+      =                       sectsz=512   attr=2, projid32bit=1
+      =                       crc=0        finobt=0
+      data     =                       bsize=4096   blocks=2097152, imaxpct=25
+      =                       sunit=1024   swidth=1024 blks
+      naming   =version 2              bsize=4096   ascii-ci=0 ftype=0
+      log      =internal log           bsize=4096   blocks=2560, version=2
+      =                       sectsz=512   sunit=8 blks, lazy-count=1
+      realtime =none                   extsz=4096   blocks=0, rtextents=0</screen>
+    </step>
+    <step>
+     <para>
+      Mount the device and check it is correctly mounted. Replace
+      <filename>/mnt</filename> with your mount point.
+     </para>
+     <screen>&prompt.root;mount /dev/rbd0 /mnt
+      &prompt.root;mount | grep rbd0
+      /dev/rbd0 on /mnt type xfs (rw,relatime,attr2,inode64,sunit=8192,...</screen>
+     <para>
+      Now you can move data to and from the device as if it was a local
+      directory.
+     </para>
+     <tip>
+      <title>Increasing the Size of RBD Device</title>
+      <para>
+       If you find that the size of the RBD device is no longer enough, you can
+       easily increase it.
+      </para>
+      <orderedlist spacing="normal">
+       <listitem>
+        <para>
+         Increase the size of the RBD image, for example up to 10 GB.
+        </para>
+        <screen>&prompt.cephuser;rbd resize --size 10000 mypool/myimage
+         Resizing image: 100% complete...done.</screen>
+       </listitem>
+       <listitem>
+        <para>
+         Grow the file system to fill up the new size of the device.
+        </para>
 <screen>&prompt.root;xfs_growfs /mnt
- [...]
- data blocks changed from 2097152 to 2560000</screen>
-      </listitem>
-     </orderedlist>
-    </tip>
-   </step>
-   <step>
-    <para>
-     After you finish accessing the device, you can unmap and unmount it.
-    </para>
+[...]
+data blocks changed from 2097152 to 2560000</screen>
+       </listitem>
+      </orderedlist>
+     </tip>
+    </step>
+    <step>
+     <para>
+      After you finish accessing the device, you can unmap and unmount it.
+     </para>
 <screen>
 &prompt.cephuser;rbd unmap /dev/rbd0
 &prompt.root;unmount /mnt
 </screen>
-   </step>
-  </procedure>
-
-  <tip>
-   <title>Manual (Un)mounting</title>
-   <para>
-    Since manually mapping and mounting RBD images after boot and unmounting
-    and unmapping them before shutdown can be tedious, an
-    <command>rbdmap</command> script and &systemd; unit is provided. Refer to
-    <xref linkend="ceph-rbd-rbdmap"/>.
-   </para>
-  </tip>
-
+    </step>
+   </procedure>
+   
+   <tip>
+    <title>Manual (Un)mounting</title>
```

Comment:
I'm not sure I like this title. Prefer the expansion of "Manual mounting and unmounting" to be consistent with other titles.

Reply:
```suggestion
    <title>Manual Mounting and Unmounting</title>